software-development-roles/software-developer/senior-developer.md

```diff
@@ -0,0 +1,153 @@
+
+
+# Senior Software Developer
+**Location:** National\*  
+**Closing Date:** [Insert Date]  
+**Interviews:** Expected w/c [Insert Week]
+
+**Grade:** G6  

```

Comment:
sorry if I'm out of the loop/this is changing, but aren't seniors grade 7?

Reply:
Thanks @carlosMoJ, I've changed the grade to G7